src/spas/libs/member-management/feature/src/lib/member-management/user-detail/user-detail.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { User } from '../user-profile';
```

Comment:
be aware that once userprofile model has been moved to domain lib, import will have to change